src/core/config.js

```diff
@@ -30,33 +31,53 @@ export default function (vm) {
       noCompileLinks: [],
       noEmoji: false,
       notFoundPage: true,
+      plugins: [],
       relativePath: false,
       repo: '',
       routes: {},
       routerMode: 'hash',
       subMaxLevel: 0,
-      themeColor: '',
+      // themeColor: '',
       topMargin: 0,
+
+      // Deprecations //////////////////
+
+      __themeColor: '',
+      get themeColor() {
+        return this.__themeColor;
+      },
+      set themeColor(value) {
+        this.__themeColor = value;
+        console.warn(
+          stripIndent(/* html */ `
+            $docsify.themeColor is deprecated. Use a --theme-color property in your style sheet. Example:
+            <style>
+              :root {
+                --theme-color: deeppink;
+              }
+            </style>
+          `).trim()
+        );
+      },
```

Comment:
If someone writes

```js
window.$docsify = {
  themeColor: 'pink'
}
```

they will see this in console:

<img width="610" alt="Screenshot 2023-07-02 at 4 53 07 PM" src="https://github.com/docsifyjs/docsify/assets/297678/edfae1f4-7b9e-4eda-a62a-30af66382226">